cogs/utils/rsstools.py

```diff
@@ -0,0 +1,75 @@
+""" A collection of functions for RSS feed parsing"""
+
+import discord
+from dateutil import parser
+import regex as re
+from __main__ import botdata
+import aiohttp
+import asyncio
+from bs4 import BeautifulSoup
+
+async def get_html(url):
+    """ Takes url, returns html. No error handling"""
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url) as response:
+            html = await response.text()
+            return html
+
+
+def has_new_dota(feed):
+    """ Takes a feedparser feed for the dota 2 blog, checks to see if newest publish date is newer
+    than what is on record. This requires looking at the botdata json field "blog_date"
+    If it cannot find the field, will automatically return false
+    returns boolean"""
+    new = ""
+    old = botdata.blog_date
+    if feed.entries:
+        new = feed.entries[0].published
+    else:
+        return False
+    if old and new != "":
+        if old != "":
+            if parser.parse(new )>parser.parse(old):
+                botdata.blog_date = new
+                return True
+            else:
+                return False
+        else:
+            botdata.blog_date = new
+            return False
```

Comment:
I think this should be `return True`?

Reply:
Possibly. This function was half baked. I am going to delete this function for now as it is not being used, then rewrite based on your suggestions when I get around to the updating task. I do appreciate the feedback for now, though.

Reply:
Sounds good